guides/docker.md

```diff
@@ -85,17 +85,17 @@ ENV LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8" LANGUAGE="en_US:en"
 
 When extracting the script or copying the default script edeliver pins the deployed version by replacing the string `{{edeliver-version}}` with the deployed version, `{{edeliver-docker-image}}` with the image name from the `RELEASE_STORE` and other tags, like `{{edeliver-app}}` with `$APP` and `{{edeliver-docker-opts}}` with `DOCKER_OPTS` from the `.deliver/config`. They can be used e.g. to **expose ports** used by the release from inside of the container to the host, e.g. `DOCKER_OPTS="--publish 8080"`.
 
 It is required, that the deploy host is also authenticated at the private docker registry. To list available tags / versions you can run `./edeliver show releases`. For a private registry `DOCKER_REGISTRY_TOKEN` must be set in the `.deliver/config` because listing tags requires authentication at the docker registry directly because listing is not supported by the docker daemon. For gcloud you can set it e.g. like this: `DOCKER_REGISTRY_TOKEN="$(gcloud auth print-access-token || :)"`, for DockerHub you can set the `DOCKER_HUB_ACCESS_TOKEN` and edeliver will generate the token for you. For any other registry you need to fetch or set the `DOCKER_REGISTRY_TOKEN` used as Bearer token in your config yourself.
 
 ## Running a Docker Image
 
-The extracted [script](https://github.com/edeliver/edeliver/blob/f5ddf03377141cad24af2e9e0fb704e2c06b2411/docker/start_container) at `$DELIVER_TO/bin/$APP` can be used to start and control the release as usual, e.g. to start the container in the foreground run `$DELIVER_TO/bin/$APP console` or to connect to a running node `$DELIVER_TO/bin/$APP remote_console`. This script **[configures](https://github.com/edeliver/edeliver/blob/f5ddf03377141cad24af2e9e0fb704e2c06b2411/docker/start_container#L80-L107) the container and the erlang / elixir node** to be able to run in a container and to connect to a running node in the container. 
+The extracted [script](https://github.com/edeliver/edeliver/blob/master/docker/start_container) at `$DELIVER_TO/bin/$APP` can be used to start and control the release as usual, e.g. to start the container in the foreground run `$DELIVER_TO/bin/$APP console` or to connect to a running node `$DELIVER_TO/bin/$APP remote_console`. This script **[configures](https://github.com/edeliver/edeliver/blob/master/docker/start_container#L120-L158) the container and the erlang / elixir node** to be able to run in a container and to connect to a running node in the container.
```

Comment:
@NexPB Thanks for updating the links to point to the most recent versions. When referencing the lines here, I think a permalink with a fixed ref makes more sense to ensure the lines are permanently correct, which would not be the case, if the most recent version has changes (above).

Reply:
Changed that link to use the commit ref of the last time the file was modified.